### ISSUE TITLE
Layout supposed to work on mobile?

### ISSUE THREAD

**Issue by mbmerrill** [vitmalina/w2ui#785]
Is the layout component supposed to work on mobile devices?  Even when I make the splitter bars bigger (to make them easier to touch), I cannot move the splitters.

**Comment by aadrian** [comment]
I suppose not really, since there's a separate project targeting mobile:
https://github.com/vitmalina/Web-2.0-Touch

vote for:
https://github.com/vitmalina/Web-2.0-Touch/issues/24   :)

**Comment by mbmerrill** [comment]
Well, I saw that project.  Last commit was two years ago, and the urls listed on the project page are for sale.  Conclusion: dead project.  So I was wondering if maybe layout had been designed for mobile instead.

**Comment by vitmalina** [comment]
W2UI project is for desktops. It will mostly work on mobile (preferably large screens), but due to many reasons it is not well suited for mobile. I can not think of a good way to make it work as all that I tried with mobile where not "good enough". For touch screens it should be different large buttons and grid does not really help here. Layout and resize should be easier and on mobile it is hard to use layout. It should also be more responsive as mobile processors are slower and grid, layout, etc. require a lot of processing power. As such I do not think that w2ui will even be on mobile something I could be proud of.

Web 2.0 Touch is an old project now and has not been updated in a while (I have updated url links). I have worked on some parts of it that I did not commit, but it was not finished. I am planning to give it some attention, but it will not happen any time soon. Thank you.
